content/jest.setupFilesAfterEnv.ts

```diff
@@ -18,3 +29,46 @@ if (process.env.LOG_LEVEL === 'off') {
     jest.spyOn(process.stdout, 'write').mockImplementation(() => true)
   })
 }
+
+async function setupApiCoverage() {
+  const coverageDir = path.join(__dirname, 'api-coverage')
+  const coverageFilePath = path.join(coverageDir, 'api-coverage.json')
+  const coverage: ApiCoverage = JSON.parse(fs.readFileSync(coverageFilePath).toString())
+
+  // Hacky way of adding a middleware to keep track of the requests without changing the Server file
+  const registerRoute = Server.prototype['registerRoute']
+  Server.prototype['registerRoute'] = async function (this: Server, ...args) {
+    if (!this['_API_COVERED_']) {
+      this['_API_COVERED_'] = true
+      this['app'].use((req, res, next) => {
+        res.on('finish', () => {
+          for (const apiPath in coverage) {
+            // Convert OpenAPI paths to a format that is testable against the server API
+            // Eg: /contents/{hashId}/active-entities -> /contents/[^\\/]*/active-entities
+            // This allows testing against any resource path like /contents/Qws4djflKjf9dJsa/active-entities
+            const replacedPath = apiPath
+              .replace(/\{.*?\}/, '[^\\/]*')
+              .replace(/\{.*?\}/, '[^\\/]*') // Replace a second time for URLs with two query params
```

Comment:
nit: `replaceAll` won't work?

Reply:
Unfortunately `replaceAll` is not available for our target lib `es2020`. We would have to update it to `es2021`, but I would do it separately to avoid any side effects